app/src/main/java/com/kamwithk/ankiconnectandroid/routing/RouteHandler.java

```diff
@@ -52,10 +58,22 @@ public NanoHTTPD.Response get(RouterNanoHTTPD.UriResource uriResource, Map<Strin
             e.printStackTrace();
         }
 
-        NanoHTTPD.Response rep = apiHandler.chooseAPI(files.get("postData"), session.getParameters());
+        NanoHTTPD.Response rep;
+        if (files.get("postData") == null) {
+            rep = newFixedLengthResponse("Ankiconnect Android is running.");
+        } else {
+            rep = apiHandler.chooseAPI(files.get("postData"), session.getParameters());
+        }
```

Comment:
Can you make sure this doesn't inadvertently affect Forvo audio? If memory serves me correct, I tried to add this in a different PR, but removed it because the Forvo audio was no longer getting fetched.

P.S. Thanks for the PR! I'll try to make some time to review it properly sometime in the next few days.

Reply:
Sure! I'll set up my yomichan according to the instructions in the readme.

No rush with the PR, take your time!

Reply:
You were right - forvo audio was broken by this change. I think I was able to fix it, though: see 87c2ecd7c034e66734915b2ea3d76c01da671128

In yomichan, I can now select all these audios and hear the word being read by different people
![image](https://user-images.githubusercontent.com/300791/227704369-458c5f78-cc32-4ae0-a198-61d2605c48c8.png)

Reply:
Looks good, thanks for the change! I'll temporarily keep this open as a self reminder to verify that it works on my end.

Reply:
Just noticed I had accidentally broken my cors support with the fix. :)
Pushed some more changes - now forvo works locally and the cors headers are also applied correctly.

Reply:
Seems like everything here works, thanks!